### ISSUE TITLE
Voiceover does not announce lift/drop after tabbing outside web-content

### ISSUE THREAD

**Issue by edwardsCam** [atlassian/react-beautiful-dnd#879]
Hey guys, loving the library so far! Accessible drag n drop has been a pain point at my org for a long time, and react-beautiful-dnd has solved many of our problems. However, I think I found an issue when using Safari + Voiceover.

## Bug or feature request?

Bug

### Expected behavior

Any time I pick up a draggable, move it, drop it, or cancel, the screenreader should make an announcement.

### Actual behavior

The announcements do work at first, and it is beautiful! However, if you tab through a complete cycle (which brings you out of the web content and onto the browser's ui, then back into the content), it stops making announcements when you pick up, move, drop, or cancel a drag.

### Steps to reproduce

- Use latest Safari + Voiceover
- Have a simple list (I used the example page at https://react-beautiful-dnd.netlify.com/iframe.html?selectedKind=single%20vertical%20list&selectedStory=basic, but I can also reproduce this on my own project)
- Tab to an item, verify that everything announces correctly (picking up, moving, dropping, cancelling)
- Now tab forward until you leave the web-content (voiceover will announce "Leaving web content", and will focus on Safari's UI)
- Tab back to a draggable item
- Hit space to start a drag

(ERROR)
- I do not get an announcement that I've picked up the item.
- When I move the item, I do not get any announcements.
- When I drop the item or cancel movement, I do not get any announcements.

### What version of `React` are you using?

16.3.2

### What version of `react-beautiful-dnd` are you running?

9.0.2

### What browser are you using?

Safari ( + Voiceover)

### Demo

https://i.imgur.com/JYDSh7A.gifv (Note that once I've tabbed out of the web-content and back in, voiceover gives no indication when I pick up, move, drop, or cancel a drag.)

FWIW, it appears to still update the `react-beautiful-dnd-announcement-0` region with the correct text as I'm dragging. For some reason the screen reader just isn't reading those changes.

**Comment by edwardsCam** [comment]
Very possible that it's an OS issue..

> what version of macos are you running?

High Sierra 10.13.6

> does voiceover work correctly in chrome on that version?

This issue does not occur in Chrome. The announcements are a bit more polite in Chrome than they are on Safari, though - it usually won't interrupt to announce a lift/move/drop. But it works.

> if not running mojave (latest), does safari work on that?

I don't have access to mojave right now, unfortunately (IT hasn't approved the upgrade yet). If someone else could test that would be awesome.

**Comment by edwardsCam** [comment]
Update: just tested on macOS 10.14 (Mojave), and could not reproduce this issue. So it seems to only happen on High Sierra, due to the issues with live regions that you linked.

I'm satisfied that this can be closed as a 3rd party issue, but up to you.

**Comment by treshugart** [comment]
Thanks for following up, @edwardsCam. Let us know if this ends up happening again.
